src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/UpdateNuGetConfigPackageSourcesMappings.cs

```diff
@@ -34,45 +37,126 @@ public class UpdateNuGetConfigPackageSourcesMappings : Task
         /// </summary>
         public string[] SourceBuildSources { get; set; }
 
+        public string VmrRoot { get; set; }
```

Comment:
It looks like this is only used to get the SBRP repo path.  Consider refactoring to be the SBRP src path.  IMO it would be better to have this logic encapsulated in the msbuild logic rather than within this task.

Reply:
Fixed with https://github.com/dotnet/installer/pull/19114/commits/3f02bc95f3815f096425785f6fd3e2a351888a7e